mne/io/ctf/ctf.py

```diff
@@ -217,11 +217,11 @@ def _get_sample_info(fname, res4, system_clock):
     else:
         n_trial = n_samp // res4['nsamp']
         n_omit = n_samp_tot - n_samp
-        n_samp = n_trial * res4['nsamp']
```

Comment:
I don't see ``n_trial`` used anywhere after this, and ``n_samp_tot`` is only used to [determine the size on disk](https://github.com/mne-tools/mne-python/blob/master/mne/io/ctf/ctf.py#L155).